### PR TITLE
New package: colmap-3.8

### PR DESCRIPTION
**Pull request by agausmann**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

File: srcpkgs/metis/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'metis'
+pkgname=metis
+version=5.1.0
+revision=1
+build_style=gnu-makefile
+hostmakedepends="cmake"
```
Comment (classabbyamp) [inline srcpkgs/metis/template:6]: why not just use the cmake build style?

Comment (agausmann) [reply srcpkgs/metis/template:6]: While still overriding `do_configure()`? If that's okay, then sure.

File: srcpkgs/colmap/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'colmap'
+pkgname=colmap
+version=3.8
+revision=1
+build_style=cmake
+makedepends="boost-devel eigen flann-devel freeimage-devel metis-devel \
+	glog-devel gflags-devel sqlite-devel glew-devel qt5-devel cgal-devel \
+	ceres-solver-devel liblz4-devel"
```
Comment (classabbyamp) [inline srcpkgs/colmap/template:8]:
```suggestion
makedepends="boost-devel eigen flann-devel freeimage-devel metis-devel
 glog-devel gflags-devel sqlite-devel glew-devel qt5-devel cgal-devel
 ceres-solver-devel liblz4-devel"
```

File: srcpkgs/metis/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+short_desc="Serial Graph Partitioning and Fill-reducing Matrix Ordering"
+maintainer="Adam Gausmann <adam@gaussian.dev>"
+license="Apache-2.0"
+homepage="http://glaros.dtc.umn.edu/gkhome/metis/metis/overview"
```
Comment (classabbyamp) [inline srcpkgs/metis/template:10]:
```suggestion
homepage="http://glaros.dtc.umn.edu/gkhome/metis/metis/overview"
changelog="http://glaros.dtc.umn.edu/gkhome/metis/metis/changes"
```

File: srcpkgs/metis/template (hunk 1 above)
Comment (classabbyamp) [reply srcpkgs/metis/template:6]: you shouldn't need to touch their weird makefile -- it runs cmake internally

File: srcpkgs/colmap/template [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+short_desc="Structure-from-Motion (SfM) and Multi-View Stereo (MVS) pipeline"
+maintainer="Adam Gausmann <adam@gaussian.dev>"
+license="GPL-3.0-or-later"
```
Comment (classabbyamp) [inline srcpkgs/colmap/template:11]: not correct: https://github.com/colmap/colmap/blob/dev/COPYING.txt

Comment (agausmann) [reply srcpkgs/colmap/template:11]: Oops, missed that field

File: srcpkgs/metis/template (hunk 1 above)
Comment (agausmann) [reply srcpkgs/metis/template]: The makefile does do one thing - resolve the absolute path the the `GKlib` folder.
Could have been done within CMake, but doing it ourselves isn't too bad.

**Comment by agausmann** [comment]
Closing since I am no longer using it and don't want to maintain it. If somebody else wants to pick this up, feel free.

**Pull request closed without merging**